dashboards/jfrog-artifactory/jfrog-artifactory.json

```diff
@@ -1,163 +1,214 @@
 {
-  "name": "Jfrog Artifactory",
+  "name": "Jfrog Artifactory v.2",
```

Comment:
Checking to see if we want to have this dashboard update in place (and keep the same name), or include the version in the name and have it publish as its own.

Reply:
I'm not sure what we would want to do here, but I changed it back to the original name for now.